testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/ServerAuthenticationTestCase.java

```diff
@@ -131,6 +116,7 @@ public void testDisableLocalAuthAndStartServers() throws Exception {
         // now restart the slave HC, but don't restart the servers and verify they've reconnected afterwards
         reloadHostController(master.getDomainClient(), "slave", false);
         slave.awaitHostController(System.currentTimeMillis());
+        waitUntilState(master.getDomainClient(), "slave", "main-three", "STARTED");
         verifyServerStarted(master.getDomainClient(), "slave", "main-three");
```

Comment:
Yeah, I thought about removing this, but I was feeling a bit conservative and figured it'd be a fast check anyway, so why not. You're right though, its not needed.